src/main/resources/db/migration/V0.5.37__create_obs_unit_sequence.sql

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Is there a reason the migration numbers jump from 35 to 37?

Reply:
I will renumber migrations to be sequential.